src/resolver/resolver.zig

```diff
@@ -1160,16 +1160,19 @@ pub const Resolver = struct {
         // Treating these paths as absolute paths on all platforms means Windows
         // users will not be able to accidentally make use of these paths.
         if (std.fs.path.isAbsolute(import_path)) {
+            var normalizer = ResolvePath.PosixToWinNormalizer{};
+            const abs_import_path = normalizer.resolve(source_dir, import_path);
+
             if (r.debug_logs) |*debug| {
-                debug.addNoteFmt("The import \"{s}\" is being treated as an absolute path", .{import_path});
+                debug.addNoteFmt("The import \"{s}\" is being treated as an absolute path", .{abs_import_path});
```

Comment:
this should print the user-provided path. though, we dont really show this anywhere. (i could never get the debug messages to work)